ios/iNaturalistReactNative.xcodeproj/project.pbxproj

```diff
@@ -436,6 +436,7 @@
 			);
 			name = "[CP] Check Pods Manifest.lock";
 			outputFileListPaths = (
+				"${PODS_ROOT}/Target Support Files/Pods-iNaturalistReactNative/Pods-iNaturalistReactNative-resources-${CONFIGURATION}-output-files.xcfilelist",
```

Comment:
This is causing the following build errors for me in iOS:

```
❌  error: Multiple commands produce '/Users/kueda/Library/Developer/Xcode/DerivedData/iNaturalistReactNative-bodcddsbdubblxajopponrkjzdoy/Build/Products/Debug-iphoneos/iNaturalistReactNative.app/RNCAsyncStorage_resources.bundle'
❌  error: Multiple commands produce '/Users/kueda/Library/Developer/Xcode/DerivedData/iNaturalistReactNative-bodcddsbdubblxajopponrkjzdoy/Build/Products/Debug-iphoneos/iNaturalistReactNative.app/AntDesign.ttf'
❌  error: Multiple commands produce '/Users/kueda/Library/Developer/Xcode/DerivedData/iNaturalistReactNative-bodcddsbdubblxajopponrkjzdoy/Build/Products/Debug-iphoneos/iNaturalistReactNative.app/Entypo.ttf'
```

and that list goes on. Does this need to exist? Without it I think this is working as expected.